src/main/java/controller/StudentsList.java

```diff
@@ -0,0 +1,35 @@
+package controller;
+
+import model.Student;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by aleksey.dobrovolsky on 6/3/2019.
+ */
+class StudentsList {
+
+    static List<Student> students = init();
```

Comment:
even if field is static, good practice is to hide properties ie making them private, and provide getter to access it.
This prevent not relevant changes of the attribute

Reply:
done - making students private + add getter

Reply:
done - changed the logic of StudentsList class. Removed static content from the class, all methods change attributes related to the objects of the class. 
students getter became redundant